### PR TITLE
VIMAND-160 Add featured content request

### PR DESCRIPTION
**Pull request by EStepiuk**
# Summary
https://vimean.atlassian.net/browse/VIMAND-160

## Description
Add featured content request.

### CODE REVIEW

File: models/src/main/java/com/vimeo/networking2/FeaturedContent.kt [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ * 
+ * @param video Featured video.
+ */
+@JsonClass(generateAdapter = true)
+data class FeaturedContent(
+    
+    @Json(name = "video")
+    val video: Video?,
+    
+)
```
Comment (RomanMinenok) [inline models/src/main/java/com/vimeo/networking2/FeaturedContent.kt]: new line is missing

File: models/src/main/java/com/vimeo/networking2/FeaturedContent.kt [hunk 2]
```diff
@@ -0,0 +1,18 @@
+package com.vimeo.networking2
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
```
Comment (RomanMinenok) [inline models/src/main/java/com/vimeo/networking2/FeaturedContent.kt:5]: double empty line

Comment (EStepiuk) [reply models/src/main/java/com/vimeo/networking2/FeaturedContent.kt:5]: resolved

File: models/src/main/java/com/vimeo/networking2/FeaturedContent.kt (hunk 1 above)
Comment (EStepiuk) [reply models/src/main/java/com/vimeo/networking2/FeaturedContent.kt]: resolved

**Review by RomanMinenok — APPROVED** [review]

**Pull request merged**